### PR TITLE
[Tests-Only][CI] Cancel Old running builds

### PR DESCRIPTION
**Pull request by jasson99**
https://github.com/owncloud/QA/issues/674
- Cancel Old running builds of a PR
**note** this PR was generated by a automated script

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -75,7 +75,7 @@ def main(ctx):
     return before + coverageTests + afterCoverageTests + nonCoverageTests + stages + after
 
 def beforePipelines(ctx):
-    return codestyle(ctx) + jscodestyle(ctx) + phpstan(ctx) + phan(ctx) + phplint(ctx) + checkStarlark()
+    return codestyle(ctx) + jscodestyle(ctx) + checkForRecentBuilds(ctx) + phpstan(ctx) + phan(ctx) + checkStarlark()
```
Comment (phil-davis) [inline .drone.star:78]: phplint needs to be put back

File: .drone.star [hunk 2]
```diff
@@ -1878,62 +1930,3 @@ def checkStarlark():
             ],
         },
     }]
-
-# Updater specific code
-# this is extra bit of code from the standard .drone.star from activity app
-def phplint(ctx):
```
Comment (phil-davis) [inline .drone.star:1884]: phplint needs to be put back

File: .drone.star (hunk 1 above)
Comment (jasson99) [reply .drone.star]: fixed

File: .drone.star (hunk 2 above)
Comment (jasson99) [reply .drone.star]: fixed

**Review by phil-davis — APPROVED** [review]
It looks like this repo was already missing some recent .drone.star changes. It's good to have the extra changes that are here, that apply the minor starlark formatting.

**Pull request merged**